recipes/r-lisreltor/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = '0.1.4' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-lisreltor
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/lisrelToR_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/lisrelToR/lisrelToR_{{ version }}.tar.gz
+  sha256: e58ecdb5c8fcf91dd1679750aabdd946973d9c5dfeaa7083553d42383ec02c7e
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+
+  host:
+    - r-base
+
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('lisrelToR')"           # [not win]
+    - "\"%R%\" -e \"library('lisrelToR')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=lisrelToR
+  license: GPL-2.0-or-later
```

Comment:
```suggestion
  license: GPL-2.0-only
```

Reply:
Thanks for reviewing all these @cbrueffer I'll try to be more diligent about verifying licenses in the future!

Reply:
No worries. Please also look at tidying up empty lines in future recipes; what `conda_r_skeleton_helper` produces is pretty good.